### DISCUSSION TITLE
BestDefaultProgram Wire1 Compiling problems

### DISCUSSION THREAD

**Discussion by Leg-154** [gizmo-platform/gizmo#15] [Q&A] [answered]
Hello! I have been attempting to load BestDefaultProgram onto my board, and I have verified that my board is able to receive and run basic Arduino programs, but I have encountered some issues when compiling. 

![image](https://github.com/user-attachments/assets/319f37d7-12d5-4522-b822-513bcec29743)

I also attempted to change Wire1 to just "Wire", but then the following issues were caused. 

![image](https://github.com/user-attachments/assets/b1a01239-919c-46a6-ad5a-bf96d4313ad0)

Thanks in advance!

**Comment by barulicm** [comment, marked as answer]
This error pops up most commonly when you're using the wrong board support package. There are multiple support packages for the Pico, but only one is supported by the Gizmo Arduino library.

Can you confirm in the board manager that you have the Pi Pico board package form Earle Philhower installed? You can check this by opening the board manager, searching for "Earle", and making sure you see an entry like this with the option to remove it. (If you see "install" instead of "remove", you do not have the board package installed.)

![image](https://github.com/user-attachments/assets/f41b5b73-407a-48f8-a945-db316d4773c9)

You will also need to make sure you have selected the correct board when you compile the sketch. You can set this by selecting it from these menus:

Tools -> Board: ... -> Raspberry Pi Pico/RP2040 -> Raspberry Pi Pico

**Comment by Leg-154** [comment]
I had the Earle Philhower version installed, but I believe it was that very last instruction that caused the problem. The board did indeed take the code now and compile correctly, thank you very much!

I am a little rusty with the modern Arduino IDE, so this helped very much.
